modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -222,26 +224,55 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 List<CPDefinitionSpecificationOptionValue> cpDefinitionSpecificationOptionValues = cpContentHelper.getCPDefinitionSpecificationOptionValues(cpDefinitionId);
 List<CPMedia> cpMedias = cpContentHelper.getCPMedias(cpDefinitionId, themeDisplay);
 List<CPOptionCategory> cpOptionCategories = cpContentHelper.getCPOptionCategories(company.getCompanyId());
+String description = cpCatalogEntry.getDescription();
+boolean hasCPDefinitionSpecificationOptionValues = cpContentHelper.hasCPDefinitionSpecificationOptionValues(cpDefinitionId);
+String navCPMediaId = "navUnderlineCPMedia";
+String navDescriptionId = "navUnderlineDescription";
+String navSpecificationId = "navUnderlineSpecifications";
```

Comment:
Add space here, we usually don't group strings and list together. If we do they should be sorted alphabetically.
```suggestion
List<CPOptionCategory> cpOptionCategories = cpContentHelper.getCPOptionCategories(company.getCompanyId());

String description = cpCatalogEntry.getDescription();
boolean hasCPDefinitionSpecificationOptionValues = cpContentHelper.hasCPDefinitionSpecificationOptionValues(cpDefinitionId);
String navCPMediaId = "navUnderlineCPMedia";
String navDescriptionId = "navUnderlineDescription";
String navSpecificationId = "navUnderlineSpecifications";
```